docs/_partials/page.hbs

```diff
@@ -61,7 +61,7 @@
           </div>
 
           <div class="header-title float-xs-right">
-            <span class="label label-primary">{{package.version}}</span>
+            <a class="label label-primary" href="https://github.com/axa-group/web-toolkit/releases" >{{package.version}}</a>
```

Comment:
Could you put this URL to the package.json? You could add a new key like `releasesUrl`.